pkgs/development/python-modules/maestral/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchFromGitHub
+, python
+, sdnotify
+, systemd
+}:
+python.pkgs.buildPythonApplication rec {
+  pname = "maestral";
+  version = "1.1.0";
+
```

Comment:
```suggestion
```

Reply:
Done. Also removed the whitespace from the maestral-qt package.